src/demo/share/jfc/SwingSet2/SwingSet2.java

```diff
@@ -848,7 +848,14 @@ private void updateThisSwingSet() {
         if (frame == null) {
             SwingUtilities.updateComponentTreeUI(this);
         } else {
+            if (currentLookAndFeel.name.contains("GTK")) {
+                this.setPreferredSize(new Dimension(PREFERRED_WIDTH + 260, PREFERRED_HEIGHT + 230));
```

Comment:
It seems height increase by further 230 is too much causing too much space below JInternalFrames which again spoils the aesthetics..Guess +80 is good enough...so 980x720 as compared to 720x640

Reply:
Updated.